salt/grains/fibre_channel.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+'''
+Grains for Fibre Channel WWN's. On Windows this runs a PowerShell command that
+queries WMI to get the Fibre Channel WWN's available.
+
+.. versionadded:: 2018.3.0
+
+To enable these grains set ``fibre_channel_grains: True``.
+
+.. code-block:: yaml
+
+    fibre_channel_grains: True
+'''
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+import glob
+import logging
+
+# Import Salt libs
+import salt.modules.cmdmod
+import salt.utils.platform
+import salt.utils.files
+
+__proxyenabled__ = ['fibre_channel']
+__virtualname__ = 'fibre_channel'
+
+# Get logging started
+log = logging.getLogger(__name__)
+
+
+def __virtual__():
+    if __opts__.get('fibre_channel_grains', False) is False:
```

Comment:
Is this setting documented anywhere? I know it's documented in this file, but i mean in other places like config options, release notes, etc.

Reply:
No

Reply:
Also documented